apps/web/app/api/escrow/dispute/evidence.ts

```
@@ -0,0 +1,47 @@
+                    url,
+                    description: description ?? null,
+                    created_at: new Date().toISOString(),
+                },
+            ])
+            .select()
+            .single();
+        if (error) {
+            return NextResponse.json({ error: error.message }, { status: 500 });
+        }
+        return NextResponse.json({ evidence: data }, { status: 201 });
+    } catch (e: any) {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)